README.md

```diff
@@ -180,6 +180,77 @@ public class BookFilter : QueryableFilter<Book>
 }
 ```
 
+## Uni-form-item & uni-form-item-group
+`uni-form-item` is an uniform component that provides possibility to decorate an optional native `<label>`, any native or custom form control element, and an optional `<output>` message as form item. `uni-form-item-group` allows to group multiple `uni-form-item` elements into one form group with separate label and message.
```

Comment:
> uniform component that provides possibility to decorate...

Do you think it would make sense to shorten this:

Same below:

> `uni-form-item-group` allows to group multiple

> `uni-form-item-group` groups multiple
> uniform component that decorates...